apps/client/src/app/pages/checklist/checklist/checklist.component.ts

```diff
@@ -28,12 +30,13 @@ export interface TaskCharacter extends Character{
   styleUrls: ["./checklist.component.less"]
 })
 export class ChecklistComponent {
-
   public TaskFrequency = TaskFrequency;
+
   public TaskScope = TaskScope;
 
+  public forceShowHiddenCharacter$ = false;
```

Comment:
Ok I just saw your usage below, you need this to be an Observable, so here's how to do it:

instead of `false`, init it with `new LocalStorageBehaviorSubject(false)`, because you'll also want the value to be persisted accross sessions.

Then in template binding, change `<label nz-checkbox [(ngModel)]="forceShowHiddenCharacter$"></label>` for 

```html
<label nz-checkbox [ngModel]="forceShowHiddenCharacter$ | async" (ngModelChange)="forceShowHiddenCharacter$.next($event)"></label>
```

Finally, in your usage below, you need to combine latest value from checkbox with latest value from roster to produce your display:

```ts
  public roster$: Observable<TaskCharacter[]> = combineLatest([this.rawRoster$, this.forceShowHiddenCharacter$]).pipe(
    map(([roster, showHidden]) => {
      return roster.characters.map( (char) => ({
        ...char,
        className: LostarkClass[char?.class].toLowerCase(),
      })).filter(char => {
        if ( showHidden ) {
          return true;
        } else {
          return !char.isHide && true;
        }
      })
    })
  );
```